bytes/bytes.go

```diff
@@ -192,6 +193,15 @@ func (b Bytes) LineFrom(start Index) (Bytes, error) {
 	return b[start:], nil
 }
 
+// Normalize normalize this sequence of bytes.
+func (b Bytes) Normalize() (Bytes, error) {
```

Comment:
I didn't understand the purpose of this function. See also the `unquoteBytes` function.